rel/i18n/emqx_bridge_kafka.hocon

```diff
@@ -255,7 +255,11 @@ socket_nodelay.label:
 """No Delay"""
 
 authentication.desc:
-"""Authentication configs."""
+"""~Authentication configs.
+- <code>none</code>: no authentication.
+- <code>msk_iam</code>: use [MSK IAM authentication](https://docs.aws.amazon.com/msk/latest/developerguide/security-iam.html).
```

Comment:
should mention this one only applicable when running in ec2?
or with environment variables set (`AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY`) ?

Reply:
:checkered_flag: 

no need to set those, they are read from the metadata endpoint